### PR TITLE
Gallery links 

### PR DESCRIPTION
**Pull request by Erin007**
Signed out users will now have a link to the public Project Gallery in the teal bar on Pegasus and Dashboard; this link will jump into the hamburger menu on small screens. 

Similarly, teachers and students will have a link to their version of the Project Gallery (studio.code.org/projects)* in the header that hops into the hamburger on small sizes. 

* The link is the same for students and teachers, but I assume the student link will go to their personal gallery.  Where does the teacher's link go? To their classroom gallery? If so, how do we know which section? I assume the Project Gallery folks handle this? @poorvasingal 
 
Example hamburger menu for a student: 
screen shot 2017-05-19 at 2 02 27 pm

### CODE REVIEW

**Comment by Erin007** [comment]
@joshlory

**Comment by poorvasingal** [comment]
The classroom gallery isn't getting built until later. Anyone who is signed in should go to studio.code.org/projects. Signed out should go to studio.code.org/projects/public. @davidsbailey - Can you double check that this is correct in terms of the links?

**Comment by Erin007** [comment]
Thanks for the clarification @poorvasingal! Those are the links I have from Tanya. I was just overthinking it.

File: shared/haml/hamburger.haml [hunk 1]
```diff
@@ -3,17 +3,22 @@
...
     { title: "Professional learning", url: "#{studio_url_base}/my-professional-learning" }
   ].freeze
 
   student_entries = [
     { title: "Home", url: "#{studio_url_base}" },
     { title: "Courses", url: "#{studio_url_base}/courses" },
+    { title: "Project Gallery", url: "#{studio_url_base}/projects" }
+  ]
+
+  not_signed_in_entries = [
```
Comment (breville) [inline shared/haml/hamburger.haml]: signed_out_* ? :)

File: shared/haml/hamburger.haml [hunk 2]
```diff
@@ -70,6 +78,8 @@
       show_teacher_options = "show-mobile"
     elsif user_type == "student"
       show_student_options = "show-mobile"
+    else
+      show_not_signed_in_options = "show-mobile"
```
Comment (breville) [inline shared/haml/hamburger.haml]: Given these options can now appear, do we now need to add them to the code below that determines if/when we show the hamburger at all?

**Comment by davidsbailey** [comment]
yep, the links look good in what @poorvasingal said and also in @Erin007 's code.

File: shared/haml/hamburger.haml (hunk 1 above)
Comment (Erin007) [reply shared/haml/hamburger.haml]: ay yi yi. changing now.

**Pull request merged**